INTERESTED_DEVELOPERS.md

```diff
@@ -55,6 +55,9 @@ If you want to be listed here, open a PR with your information, just order yours
 * @sungam3r
   * Company/Project/Repo: https://github.com/graphql-dotnet/server
   * Reason: Interested in client/server spec
+* @glennblock
+  * Company/Project/Report: https://github.com/microsoft
+  * Reason: Interested in a common HTTP spec, and in adoption of emerging HTTP standards like HTTP SEARCH
```

Comment:
Please move this up so it's in alphabetical order by username :+1:

Reply:
Hi @benjie, I will fix and send a new PR